modules/configuring-a-floating-action-button/proc-configuring-floating-action-button-as-a-dynamic-plugin.adoc

```diff
@@ -16,51 +16,127 @@ To configure a floating action button as a dynamic plugin, complete any of the f
+* To configure a dynamic plugin as a floating action button that opens an external link, specify the `global.floatingactionbutton/config` mount point in your dynamic plugins config map within the `backstage-plugin-global-floating-action-button` plugin. For example:
 +
 .Example of a floating action button that opens GitHub
 [source,yaml]
 ----
-  dynamicPlugins:
-    frontend:
-      red-hat-developer-hub.backstage-plugin-global-floating-action-button:
-        mountPoints:
-          - mountPoint: application/listener
-            importName: DynamicGlobalFloatingActionButton
-          - mountPoint: global.floatingactionbutton/config
-            importName: NullComponent
-            config:
-              icon: github
-              label: 'Git'
-              toolTip: 'Github'
-             	to: https://github.com/redhat-developer/rhdh-plugins
+- package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-global-floating-action-button
+  disabled: false
+  pluginConfig:
+    dynamicPlugins:
+      frontend:
+        red-hat-developer-hub.backstage-plugin-global-floating-action-button:
+          mountPoints:
+              - mountPoint: application/listener
+                importName: DynamicGlobalFloatingActionButton
+              - mountPoint: global.floatingactionbutton/config
+                importName: NullComponent # <1>
+                config:
+                  icon: '<svg viewBox="0 0 250 300" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid"><path d="M200.134 0l55.555 117.514-55.555 117.518h-47.295l55.555-117.518L152.84 0h47.295zM110.08 99.836l20.056-38.092-2.29-8.868L102.847 0H55.552l48.647 102.898 5.881-3.062zm17.766 74.433l-17.333-39.034-6.314-3.101-48.647 102.898h47.295l25-52.88v-7.883z" fill="#40B4E5"/><path d="M152.842 235.032L97.287 117.514 152.842 0h47.295l-55.555 117.514 55.555 117.518h-47.295zm-97.287 0L0 117.514 55.555 0h47.296L47.295 117.514l55.556 117.518H55.555z" fill="#003764"/></svg>' # <2>
+                  label: 'Quay'
+                  showLabel: true
+                  toolTip: 'Quay'
+                  to: 'https://quay.io'
+              - mountPoint: global.floatingactionbutton/config
+                importName: NullComponent
+                config:
+                  icon: github
+                  label: 'Git'
+                  toolTip: 'Github'
+                  to: https://github.com/redhat-developer/rhdh-plugins
 ----
+<1> (Required) The import name with an associated component to the mount point.
+<2> Use this `svg` value to display the `Quay` icon.
+
+* To configure a floating action button that contains a submenu, define the `global.floatingactionbutton/config` mount point in the same `slot` field in your dynamic plugins config map for multiple dynamic plugins as actions. The default slot is `page-end` when not specified. For example:
```

Comment:
```suggestion
* To configure a floating action button that contains a submenu, define the `global.floatingactionbutton/config` mount point in the same `slot` field in your dynamic-plugins config-map for multiple actions. The default slot is `page-end` when not specified. For example:
```

Can you check what is the correct way to write the terms "dynamic plugins" and "config map"
should they have '-' between the words